YT_CRUD/.env.dist

```diff
@@ -8,3 +8,10 @@ APP_SECRET=c210d587b4c1711434e345098f593a60
 #TRUSTED_PROXIES=127.0.0.1,127.0.0.2
 #TRUSTED_HOSTS=localhost,example.com
 ###< symfony/framework-bundle ###
+
+###> doctrine/doctrine-bundle ###
+# Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
+# Configure your db driver and server_version in config/packages/doctrine.yaml
+DATABASE_URL=mysql://db_user:db_password@127.0.0.1:3306/db_name
```

Comment:
@JanMaria Mam nadzieję, że skopiowałeś ten plik do pliku o nazwie `.env` i podmieniłeś `DATABASE_URL` swoimi danymi dostępowymi. Nazwa użytkownika to **root**, hasło też **root**. Jeżeli nie wiesz jak utworzyć bazę danych to jest kilka sposobów. Poszukaj imformacji jak utworzyć bazę danych przy wykorzystaniu jednej z komend z listy widocznej po wpisaniu w terminalu `bin/console`. Jeżeli nie utworzyłeś pliku `.env` to dowiedz się czym jest ten plik i za co odpowiada.